
# The goal state for the model to plan for. The state is not necessarily explicit.
# The goal is considered achieved if all the constraints are satisfied.
# An explicit state can be specified by setting joint constraints to a specific value.
motion_planning_msgs/KinematicConstraints goal_constraints

# The set of constraints that need to be satisfied by the entire solution path.
motion_planning_msgs/KinematicConstraints path_constraints

# The set of allowed contacts
motion_planning_msgs/AcceptableContact[] contacts
---
---
int32 mode
int32 PLANNING=1
int32 MOVING=2

duration time_to_completion
